protocol-dashboard/src/utils/format.ts

```diff
@@ -177,11 +177,18 @@ export const getShortDate = (ms: number) => {
   const date = new Date(ms)
   const month = date.getUTCMonth()
   const day = date.getUTCDate()
-  return `${day} ${months[month]}`
+  return `${months[month]} ${day}`
 }
 
 export const getShortMonth = (ms: number) => {
   const date = new Date(ms)
   const month = date.getUTCMonth()
   return months[month]
 }
+
+export const getShortMonthAndYear = (ms: number) => {
```

Comment:
is there a way to use dayjs for parsing and formatting? might be easier to read and maintain if that is possible.